saleor/graphql/shop/schema.py

```diff
@@ -40,9 +45,16 @@ class ShopQueries(graphene.ObjectType):
     def resolve_shop(self, _info):
         return Shop()
 
-    @load_site_callback
-    def resolve_order_settings(self, _info, site):
-        return site.settings
+    def resolve_order_settings(self, _info):
+        channel = channel_models.Channel.objects.order_by("name").first()
```

Comment:
I think `slug` instead of `name` would be better. It is unique and `name` can be a just empty string or something.